pkg/client/client.go

```diff
@@ -199,4 +214,38 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse) {
 			}
 		}
 	}
+
+	if sr.Controllers != nil {
+		nFailing, out := 0, []string{"  Name\tLast success\tLast error\tCount\tMessage\n"}
+		for _, ctrl := range sr.Controllers {
+			if status := ctrl.Status; status != nil {
+				if status.ConsecutiveFailureCount == 0 {
+					if !allControllers {
+						continue
+					}
+				} else {
+					nFailing++
+				}
```

Comment:
I found this logic a bit difficult. Why not:
```golang
if status.ConsecutiveFailureCount > 0 {
	nFailing++
} else if !allControllers {
	continue
}
```

or a switch on the same clauses.